_includes/head.html

```diff
@@ -1,8 +1,8 @@
 <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Connecting open source & design! 🎨" />
     <meta name="author" content="">
```

Comment:
There are still some metas opened. :) So you've missed a few. 
These fixes are not mandatory for this PR, but it is nice from a consistency perspective.

Reply:
Oops!  I'll finish that up.

Reply:
Done.  All `meta` tags and `link` tags are closed with `/>`.  I also added the keywords meta tag for SEO but other then that I think everything is taken care of.